### ISSUE TITLE
Avaliação da primeira entrega

### ISSUE THREAD

**Issue by mecvamecva** [diegortx/TrabalhoIV#1]
Estou avaliando o trabalho com 100%, porém tenho duas observações abaixo.

Observação 1: Não faz o menor sentido enviar o arquivo compactado para o repositório do Git. Perde todo o sentido do uso da ferramenta de controle de versões. Para esta entrega estou aceitando, mas na próxima não irei aceitar.

Observação 2: Diego Nogueira, você enviou e-mail do repositório com cópia para o Jonathan. O que significa isso? Ele fará o trabalho com vocês? Se sim, alinhe com ele pois ele me enviou o repositório dele em um e-mail separado. Além disso, o trabalho é em grupo de no máximo dois e o Diego Santos também está em cópia. Por isso estou considerando como dois trabalhos separados. Para este repositório estou considerando somente os dois Diegos.

**Comment by diegortx** [comment]
Sim, Marcio são 2 trabalhos separados, enviei uma copia para ele sim, para ele ver como estamos fazendo mais ele esta fazendo sozinho, e sim irei providenciar para que os próximos sejam enviados separadamente no controle de versões, coloquei como .rar pois tinham muitos arquivos, mais lembrei que no git pode colocar arquivos separados. Obrigado
